tests/sqllogictests/suites/duckdb/join/semianti/antijoin.test

```diff
@@ -35,10 +35,8 @@ SELECT * FROM left_table ANTI JOIN (SELECT a as foo from right_table where b = 1
 43	1	1
 
 # Should throw error when filtering on column in right table
-statement error
+statement error (?s)1065.*column a doesn't exist
```

Comment:
What's `(?s)`?

Reply:
It is used to fetch the multiline error `column a doesn't exist`. The original error info is as follows:

```
        mysql client error: Server error: `ERROR HY000 (1105): SemanticError. Code: 1065, Text = error: 
  --> SQL:1:98
  |
1 | SELECT * FROM left_table ANTI JOIN right_table ON left_table.a = right_table.a WHERE right_table.a < 43;
  |                                                                                                  ^ column a doesn't exist

.'

```

Reply:
oh, it's a regex syntax.